src/Bundle.ts

```diff
@@ -53,7 +53,11 @@ export default class Bundle implements BundleInterface {
   public getIds(): string[] {
     const ids = [];
     for (let i = HEADER_START; i < HEADER_START + 64 * this.length; i += 64) {
-      ids.push(base64url.encode(this.binary.subarray(i + 32, i + 64)));
+      let bundledID = this.binary.subarray(i + 32, i + 64)
```

Comment:
@angrymouse could you use `bundleId` pls and also `const`

Reply:
@joshbenaron Done, however there context is about IDs bundled in bundle